frontend/src/components/Comment/Comment.js

```diff
@@ -1,21 +1,42 @@
-import React from 'react'
-import "./Comment.scss";
-import {Link} from "react-router-dom";
+import React, {useState, useEffect} from 'react'
+import { Link } from "react-router-dom"
+import './SCSS/Comment.scss'
+export const Comment = (props) => {
+    const [timePosted, setTimePosted] = useState(0);
+    const [timeFormat, setTimeFormat] = useState("sec");
+    
+    useEffect(() => {
+        timeElapseSincePosted();
+    }, [])
+    const timeElapseSincePosted = () => {
+        let currentDate = new Date();
+        let timePosted = currentDate.getTime() - new Date(props.comment.created).getTime();
+        timePosted = (timePosted / 1000).toFixed(0);
 
-class Comment extends React.Component {
-    //todo: add like on comments
-    render() {
-        return (
-            <div className="comment-div">
-                <br/>
-                <Link to={`/account/${this.props.comment.account}`}>{this.props.comment.account}</Link>: {this.props.comment.comment}
-                <div className="date-created">{new Date(this.props.comment.created).toUTCString()}</div>
-                <br/>
-            </div>
-        )
+        if(timePosted < 3600){
+            //Display in minutes
+            timePosted = (timePosted/60).toFixed(0);
+            setTimeFormat("min");
+        }
+        else if(timePosted <86400){
+            //Display in hours
+            timePosted = (timePosted/(60*60)).toFixed(0);
+            setTimeFormat("h");
+        }
+        else if(timePosted >= 86400){
+            //Display in days
+            timePosted = (timePosted/(60*60*24)).toFixed(0);
+            setTimeFormat("days");
+        }
```

Comment:
i think we should include seconds, months, and years in addition to the current minutes, hours, and days

Reply:
i see that by default `timeFormat` is "sec" but it seems like it will never stay as "sec" (will automatically change to something else)

Reply:
When a comment has just been posted, it will be shown in a second. (within the first minute). And you're right, I forgot to add the condition for seconds.